cmd/root.go

```diff
@@ -92,30 +91,44 @@ func checkVersionExists(versionUrl string) bool {
 
 // getRemoteVersion : From public container get the latest blobfuse version
 func getRemoteVersion(req string) (string, error) {
-	resp, err := http.Get(req)
+	// 'release/latest' page redirects URL to latest release tag
```

Comment:
This code won't work if we just released a blobfuse v1 patch right?

Reply:
we do not see any clear solution for this problem.

Reply:
either there is api to get the latest version string or there is api to list all tags/releases which we need to sort in code in descending order and figure out the latest v2 release.